Build/Projects/MonoGame.Tests.definition

```diff
@@ -509,6 +509,9 @@
     <Content Include="Assets\Xml\12_MathTypes.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+	<Content Include="Assets\Xml\12_MathTypesOutput.xml">
```

Comment:
You know I think a better idea here is to rename this like `21_CustomFormatting.xml`.  And have a test focus on custom/alternate formatting during deserialization for different types.